MetadataMigration/src/test/java/org/opensearch/migrations/commands/MigrateTest.java

```diff
@@ -0,0 +1,37 @@
+package org.opensearch.migrations.commands;
+
+import org.junit.jupiter.api.Test;
+
+import org.opensearch.migrations.MetadataArgs;
+import org.opensearch.migrations.MetadataMigration;
+import org.opensearch.migrations.metadata.tracing.RootMetadataMigrationContext;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Mockito.mock;
+
+class MigrateTest {
+
+    @Test
+    void migrate_failsInvalidInvalidParameters() {
+        var args = new MetadataArgs();
+        var context = mock(RootMetadataMigrationContext.class);
+        var meta = new MetadataMigration(args);
+
+        var configureSource = meta.migrate().execute(context);
+
+        assertThat(configureSource.getExitCode(), equalTo(999));
+    }
+
+    @Test
+    void migrate_failsUnexpectedException() {
+        var args = new MetadataArgs();
+        args.fileSystemRepoPath = "";
+        var context = mock(RootMetadataMigrationContext.class);
+        var meta = new MetadataMigration(args);
+
+        var configureSource = meta.migrate().execute(context);
+
+        assertThat(configureSource.getExitCode(), equalTo(888));
```

Comment:
the 999 and 888 look fairly magical/arbitrary from here, maybe worth comparing against the constant that defines it?

Reply:
Ah, I just realized that they're defined privately in Migrate.

Is there a reason the exit codes aren't defined in a independent class so they can be used by all the function classes without being redefined? If not, I'd lean in that direction.